src/customActions.js

```diff
@@ -112,21 +112,21 @@ CustomActions.prototype.add = function (customAction) {
 };
 
 
-CustomActions.prototype.addScriptLink = function (name, url, scope = "Web") {
+CustomActions.prototype.addScriptLink = function (name, url, scope = "Web", sequence = 100) {
```

Comment:
4 params, with 2 of  them optional is a good hint to cutover to an options object. I think with should change the signature to .addScriptLink(name, url, opts)